tidb/src/main/java/io/tidb/bigdata/tidb/types/Converter.java

```diff
@@ -114,7 +114,7 @@ public static Long safeConvertToUnsigned(Object value, Long upperBound)
       UnsignedLong unsignedLong = stringToUnsignedLong((String) value);
       result = unsignedLong.longValue();
     } else if (value instanceof BigDecimal) {
-      result = ((BigDecimal) value).longValueExact();
+      result = ((BigDecimal) value).longValue();
```

Comment:
why change this?

Reply:
if not change,  `unsigned long` will be overflow here.